File: drills/python/harnesses/nc03_two_sum.py
try:
    (twoSum)
except NameError:
    raise Exception("__signature_mismatch__")

__results__ = []
def __case__(label, expected, actual):
    __results__.append([label, repr(expected), repr(actual)])

__case__("twoSum([2, 7, 11, 15], 9)", [0, 1], twoSum([2, 7, 11, 15], 9))
__case__("twoSum([3, 2, 4], 6)", [1, 2], twoSum([3, 2, 4], 6))
__case__("twoSum([3, 3], 6)", [0, 1], twoSum([3, 3], 6))
__case__("twoSum([1, 2], 7)", [], twoSum([1, 2], 7))

File: drills/go/harnesses/nc03_two_sum.go
package main

func main() {
	run(func() []testCase {
		return []testCase{
			tc("twoSum([2, 7, 11, 15], 9)", []int{0, 1}, twoSum([]int{2, 7, 11, 15}, 9)),
			tc("twoSum([3, 2, 4], 6)", []int{1, 2}, twoSum([]int{3, 2, 4}, 6)),
			tc("twoSum([3, 3], 6)", []int{0, 1}, twoSum([]int{3, 3}, 6)),
			tc("twoSum([1, 2], 7)", []int{}, twoSum([]int{1, 2}, 7)),
		}
	})
}
